Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -642,6 +646,10 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters,
     ScriptExecutionContext::setCrossOriginMode(parameters.crossOriginMode);
     DeprecatedGlobalSettings::setArePDFImagesEnabled(!isLockdownModeEnabled());
 
+#if HAVE(LOCKDOWN_MODE_FRAMEWORK)
+    PAL::setLockdownModeEnabledForCurrentProcess(isLockdownModeEnabled());
```

Comment:
It's a little unfortunate that the process-specific state for LDM (`isLockdownModeEnabled()`) has exactly the same label as the global `PAL::isLockdownModeEnabled()`, which is a bit confusing (but not a problem in YOUR code change).

This is great, thank you for fixing.